app/src/main/java/openfoodfacts/github/scrachx/openfood/models/DatabaseHelper.java

```diff
@@ -78,6 +79,29 @@ private void upgrade(Database db, int migrateVersion) {
                 CategoryNameDao.createTable(db, true);
                 break;
             }
+            case 7: {
+                //  db.execSQL("ALTER TABLE additive_name ADD COLUMN 'wikiDataId' TEXT NOT NULL DEFAULT '';");
+                //db.execSQL("ALTER TABLE additive_name ADD COLUMN 'isWikiDataIdPresent' INTEGER NOT NULL DEFAULT '';");
+                AdditiveDao.dropTable(db,true);
+                AdditiveDao.createTable(db, true);
+
+                AdditiveNameDao.dropTable(db,true);
+                AdditiveNameDao.createTable(db, true);
+
+                CategoryNameDao.dropTable(db,true);
+                CategoryNameDao.createTable(db, true);
+
+                CategoryDao.dropTable(db,true);
+                CategoryDao.createTable(db, true);
+
+                LabelNameDao.dropTable(db,true);
+                LabelNameDao.createTable(db, true);
+
+                LabelDao.dropTable(db,true);
+                LabelDao.createTable(db, true);
```

Comment:
I reverted this commit. Sorry for the hasty merging.

Reply:
Actually, I tried that at first but it was throwing an error, I know something else was interfering with this but the bug was in the main branch so I had to do it fast. I thought this could be fixed later, my priority was to make app migrate smoothly for now.